src/timeline/media/js/angular-animate.min.js

```diff
@@ -1,57 +1,57 @@
 /*
- AngularJS v1.7.2
- (c) 2010-2018 Google, Inc. http://angularjs.org
+ AngularJS v1.6.0
+ (c) 2010-2016 Google, Inc. http://angularjs.org
```

Comment:
No chance we could move the main AngularJS version _up_ to 1.7.2, instead? (Still wildly out of date, but... 2 years less so.)

Also, if replacing `angular-animate.min.js`, the source map in `angular-animate.min.js.map` should either also be replaced, or it should be deleted, as it's no longer valid for the file it maps.

Reply:
My preference would be to store these in files named e.g. `angular-animate-1.6.0.min.js`, and reference them that way in the HTML. Yeah, it means we have to edit the HTML if we change versions, but... we _should_ have to edit the HTML, and possibly the JS, when we change versions! And it makes the version in use more obvious.

Reply:
We have some incompatibilities with newer versions of Angular, which is why this got downgraded in the first place. I think the next step will be to rewrite the timeline using non-web technologies... so this is a temporary solution.